.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -16,6 +16,23 @@ body:
       placeholder: Please describe how is the actual behaviour currently.
     validations:
       required: true
+        
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: To Reproduce
+      description: Steps to reproduce the behavior
+    validations:
+      required: false
+        
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: Text
+    validations:
+      required: false
```

Comment:
I think it is better to put this section on after `expected`, as this might be extremely long.